Automattic/wp-calypso
# Stats: Adding a global provider to identify internal sites

Related to #88688

## Proposed Changes

* adding a global provider to identify sites as internal or external
* adding missing translation

## Testing Instructions



* open the live branch
* go to a blog and inspect the network tab, verify that a call to `usage` endpoint returns `is_internal` property
* verify that the value from the above endpoint is correct for
  * an internal site - Simple/Atomic/Jetpack
  * an external site - Simple/Atomic/Jetpack
* verify if you see the UTM module in the above cases
* check the Traffic, post detail and UTM summary pages

While extending the summary page, I added the missing translation. When applied, the page content looks like this:
<img width="228" alt="SCR-20240326-ocis" src="https://github.com/Automattic/wp-calypso/assets/112354940/f51fb69a-f3af-4305-b4bd-4bd262abf427">


## Pre-merge Checklist
